src/ARVRInterface.cpp

```diff
@@ -113,6 +115,10 @@ godot_bool godot_arvr_initialize(void *p_data) {
 
 	// We (already) have our API instance? cool!
 	if (arvr_data->openxr_api != nullptr) {
+		// Register our extensions
+		arvr_data->openxr_api->register_extension_wrapper<XRFbDisplayRefreshRateExtensionWrapper>();
```

Comment:
While I hope nobody does this in practice, in theory someone could uninitialise the interface and re-initialise it in which case we'll double register the extension wrappers. We may want to either make sure the logic ignores wrappers that have already been registered or find another way to only execute this code on the first initialize call.

Reply:
I switched from a `std::vector` to a `std::set` to prevent extension wrappers from being registered more than once.